gulpfile.js

```diff
@@ -23,7 +23,7 @@ var oghliner = require('./index.js');
 gulp.task('default', ['build', 'offline']);
 
 gulp.task('build', function(callback) {
-  return gulp.src('app/**').pipe(gulp.dest('dist'));
+  return gulp.src('templates/app/**').pipe(gulp.dest('dist'));
```

Comment:
Now that you're no longer removing the app/ subdirectory, this should continue to build app/ rather than templates/app/.